bin/ontobio-parse-assocs.py

```diff
@@ -143,9 +143,11 @@ def main():
         rule_set = assocparser.RuleSet.ALL
     
     goref_metadata = None
+    goref_species = None
     if args.metadata_dir:
         absolute_metadata = os.path.abspath(args.metadata_dir)
-        goref_metadata = metadata.yamldown_lookup(os.path.join(absolute_metadata, "gorefs"))    
+        goref_metadata = metadata.yamldown_lookup(os.path.join(absolute_metadata, "gorefs"))
+        goref_species = metadata.yaml_set(absolute_metadata, "go-reference-species.yaml", "taxon_id")
```

Comment:
My nitpicky thing here that doesn't need to be fixed (and I'm very open to arguing whether it's actually wrong) is the naming of the "goref_species" variable. I think its name is derived from `goref_metadata` above, which points to GO_REF ID data. However, the species data is totally different from GO_REFs and I might be misled into thinking the `goref_species` var has something to do with GO_REF IDs.

Could we call it something like `reference_species_metadata` or `ref_species_metadata`? @mugitty What do you think? Ignore this if you like!

Reply:
@dustine32, I have updated the variable name

Reply:
Thanks so much @mugitty!